src/components/UIShell/UIShell.js

```diff
@@ -257,6 +259,9 @@ function UIShell({
               text="Flow Documentation"
             />
           ),
+          isFeedbackEnabled && (
+            <Feedback platformName={platformName} sendIdeasUrl="https://ideas.ibm.com" key="Feedback" />
```

Comment:
Per comment in Slack, any discussion to make this a dynamic value that comes from the service?

Reply:
Yes I just missed this and left the default value in.